packages/Webkul/Admin/src/Http/Controllers/ExportController.php

```diff
@@ -20,7 +20,7 @@ public function export()
 
         $gridName = explode('\\', $criteria['gridName']);
 
-        $gridInstance = app('\Webkul\Admin\DataGrids' . '\\' . last($gridName));
+        $gridInstance = app($criteria['gridName']);
```

Comment:
Have you tested this,
https://github.com/bagisto/bagisto/blob/master/packages/Webkul/Admin/src/Resources/views/export/export.blade.php#L12

Reply:
Ignore. My mistake.